beetsplug/beatport.py

```diff
@@ -259,7 +259,7 @@ def __init__(self, data):
         self.track_number = data.get('trackNumber')
         if 'bpm' in data:
             self.bpm = data['bpm']
-        if 'key' in data:
+        if data.get('key'):
```

Comment:
I'm unsure if the `key` dict will always contain the `shortName` key. If not, to be safe we could also rewrite this as:
```python
musical_key = data.get('key').get('shortName')
if musical_key:
	self.musical_key = six.text_type(musical_key)
```